plugins/modules/system/puppet.py

```diff
@@ -174,9 +179,7 @@ def main():
             manifest=dict(type='str'),
             noop=dict(type='bool'),
             logdest=dict(type='str', default='stdout', choices=['all', 'stdout', 'syslog']),
-            show_diff=dict(
-                type='bool', default=False, aliases=['show-diff'],
-                removed_in_version='7.0.0', removed_from_collection='community.general'),
+            show_diff=dict(type='bool', default=False, aliases=['show-diff']),
```

Comment:
We should still deprecate that alias:
```suggestion
            show_diff=dict(
                type='bool', default=False, aliases=['show-diff'],
                deprecated_aliases=[dict(name='show-diff', version='7.0.0', collection_name='community.general')])),
```

Reply:
This introduced a syntax error I fixed through b0f749d9